.circleci/config.yml

```diff
@@ -162,9 +162,6 @@ jobs:
         description: Whether to notify on failure
         type: boolean
         default: false
-      mips64:
-        type: boolean
-        default: false
```

Comment:
This was setup so we could run cannon 32 and 64 tests in a matrix. Which speeds up the `main` CI workflow as the `cannon-go-lint-and-test` job runs in parallel. Looking at the [latest build times](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/70454/workflows/871bf6e5-b967-4791-a201-05c28b889443/jobs/2905451), it takes 7 minutes to run both jobs when cached, and half of that time is evenly split between 32 and 64 bit test runs.

So I suggest keeping the matrix setup to reduce build times.

Reply:
Yeah - that makes total sense.  I just don't know what I'm doing with circleci 😂  I will fiddle with it.